packages/block-directory/src/components/downloadable-block-list-item/style.scss

```
@@ -69,6 +69,10 @@
 		align-items: center;
 		justify-content: center;
 	}
+
+	.is-installing & {
+		margin-right: $grid-unit-20 + 6px;
```

We should probably add a reason for this magic number, until we refactor to a better intrinsic layout:

```suggestion
    // Adding an extra 6px to avoid the UI from jumping when the rating bar gets hidden
		margin-right: $grid-unit-20 + 6px;
```

---

Makes sense 👍